packages/trader/src/Stores/Modules/Trading/Helpers/contract-type.ts

```diff
@@ -50,6 +50,7 @@ type TConfig = ReturnType<typeof getContractTypesConfig>[string]['config'] & {
     cancellation_range?: string[];
     barrier_choices?: string[];
 };
+type TNonAvailableContractsList = Required<ContractsForSymbolResponse>['contracts_for']['available'];
```

Comment:
```suggestion
type TNonAvailableContractsList = Required<ContractsForSymbolResponse>['contracts_for']['non_available'];
```

Reply:
@henry-deriv I had to add this type because deriv/api-types doesn't give the correct type for non_available, just `unknown`.  Anyway, thank you for the suggestion, it helped me realize that I can specify the exact type for it👍🏻

Reply:
i guess types needs to be generated again since API is updated?